src/haskell/install.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -e
+set -o xtrace
+
+GHCUP_VERSION="0.1.19.2"
```

Comment:
Yeah, please use the bootstrap script to always get the latest ghcup. Pinning ghcup version is not recommended.